### PR TITLE
Set tableView.bounce to NO

### PR DESCRIPTION
**Pull request by cnatoli**

### CODE REVIEW

**Comment by cnatoli** [comment]
cc: @andymoe, @leskowicz

File: iGCS/UserInterface/en.lproj/MainStoryboard.storyboard [hunk 1]
```diff
@@ -1310,7 +1310,7 @@
         <scene sceneID="cVi-Id-eoD">
             <objects>
                 <tableViewController id="g0h-Hk-vw4" customClass="MissionItemTableViewController" sceneMemberID="viewController">
-                    <tableView key="view" clipsSubviews="YES" contentMode="scaleToFill" alwaysBounceVertical="YES" dataMode="prototypes" style="plain" separatorStyle="default" rowHeight="44" sectionHeaderHeight="22" sectionFooterHeight="22" id="J2H-Pu-dEF">
+                    <tableView key="view" clipsSubviews="YES" contentMode="scaleToFill" bounces="NO" alwaysBounceVertical="YES" dataMode="prototypes" style="plain" separatorStyle="default" rowHeight="44" sectionHeaderHeight="22" sectionFooterHeight="22" id="J2H-Pu-dEF">
```
Comment (leskowicz) [inline iGCS/UserInterface/en.lproj/MainStoryboard.storyboard]: Hmm, I've used it so long with the bounce that it  feels unnatural to not have it.  I could go either way on this one though.

Comment (btgalush) [inline iGCS/UserInterface/en.lproj/MainStoryboard.storyboard]: +1 looks good.  To be clear the long term functionality that is desired is
bounce with the header (Seq#, Command, Latitude...) staying still.  The
header separating from the map view looks like a software glitch.

Comment (cnatoli) [inline iGCS/UserInterface/en.lproj/MainStoryboard.storyboard]: @leskowicz - my thoughts exactly. Going to let @btgalush and @andymoe arm-wrestle over the accept/decline button on this one.

Comment (andymoe) [inline iGCS/UserInterface/en.lproj/MainStoryboard.storyboard]: Let's hold off for now. I've got some mission mocks coming your way tonight
and I've not had a chance to play with this.



On Jan 24, 2015, at 5:02 PM, Claudio Natoli <notifications@github.com>
wrote:

In iGCS/UserInterface/en.lproj/MainStoryboard.storyboard
<https://github.com/fightingwalrus/iGCS/pull/159#discussion_r23500895>:

> @@ -1310,7 +1310,7 @@
>          <scene sceneID="cVi-Id-eoD">
>              <objects>
>                  <tableViewController id="g0h-Hk-vw4" customClass="MissionItemTableViewController" sceneMemberID="viewController">
> -                    <tableView key="view" clipsSubviews="YES" contentMode="scaleToFill" alwaysBounceVertical="YES" dataMode="prototypes" style="plain" separatorStyle="default" rowHeight="44" sectionHeaderHeight="22" sectionFooterHeight="22" id="J2H-Pu-dEF">
> +                    <tableView key="view" clipsSubviews="YES" contentMode="scaleToFill" bounces="NO" alwaysBounceVertical="YES" dataMode="prototypes" style="plain" separatorStyle="default" rowHeight="44" sectionHeaderHeight="22" sectionFooterHeight="22" id="J2H-Pu-dEF">

@leskowicz <https://github.com/leskowicz> - my thoughts exactly. Going to
let @btgalush <https://github.com/btgalush> and @andymoe
<https://github.com/andymoe> arm-wrestle over the accept/decline button on
this one.

—
Reply to this email directly or view it on GitHub
<https://github.com/fightingwalrus/iGCS/pull/159/files#r23500895>.

**Comment by andymoe** [comment]
Looked into this a bit and the actual fix Bryan wants is a bit more involved and requires some animation stuff (as well as this change) so at this stage I'm going to merge this in and file/update the ticket for the additional work.

**Pull request merged**